### DISCUSSION TITLE
Creating a folder with the title of the Anime

### DISCUSSION THREAD

**Discussion by Flugzahn** [crunchy-labs/crunchy-cli#339] [Ideas]
Hey,
it would be really nice if it could create a folder with the anime title where the episodes are saved

**Comment by kralverde** [comment]
You can already; with the -o flag do something like {series_name}/{season_name}/{episode}

**Reply by Flugzahn** [reply]
But it wont create a folder...

I mean like it should create a folder like "Solo Leveling" and save the episodes in it

**Reply by kralverde** [reply]
It will create a folder on Linux/mac. That's correct, the example I gave should create a folder with the name then season name then episodes inside the inner folder.

On windows you probably have to do \\

**Reply by kralverde** [reply]
This is what I use to create folders to separate shows/seasons:

`-o "{series_name}/{S{season_number} {season_name}/[E{episode_number}] {title} [{audio}] [{width}x{height}].mkv"`

**Comment by ySergi** [comment]
We should be able to create a custom output folder, something like `--folder {"C:\Users\Sergio\Videos\Anime"}`.

**Comment by bytedream** [comment]
The solution @kralverde provided is correct, there is no need for an extra flag.
If you set `-o /this/is/a/folder/{title}.mp4` (or `-o C:\this\is\a\folder\{title}.mp4` if you're on Windows), all files are placed in the folder `/this/is/a/folder`. The `{title}.mp4` suffix specifies the filename the file will be named after, `{title}` is a special option that will be replaced internally (you can read more [here](https://github.com/crunchy-labs/crunchy-cli?tab=readme-ov-file#output-template-options)).
